packages/js/product-editor/src/blocks/product-fields/variation-options/edit.tsx

```diff
@@ -57,19 +58,26 @@ export function Edit( {
 			'default_attributes'
 		);
 
-	const { postType } = context;
 	const productId = useEntityId( 'postType', postType );
 
-	const { attributes, handleChange } = useProductAttributes( {
-		allAttributes: entityAttributes,
-		isVariationAttributes: true,
-		productId: useEntityId( 'postType', 'product' ),
-		onChange( values, defaultAttributes ) {
-			setEntityAttributes( values );
-			setEntityDefaultAttributes( defaultAttributes );
-			generateProductVariations( values, defaultAttributes );
-		},
-	} );
+	const { attributes, fetchAttributes, handleChange } = useProductAttributes(
+		{
+			allAttributes: entityAttributes,
+			isVariationAttributes: true,
+			productId: useEntityId( 'postType', 'product' ),
```

Comment:
If we have the productId in line 61 why we get it here again instead of using it.

Reply:
Good call! Done in commit [d9213b4](https://github.com/woocommerce/woocommerce/pull/47361/commits/d9213b4746ef297f9303f770c41b6eb4d2dd4d6c)